packages/generators/add-generator.ts

```diff
@@ -125,44 +143,46 @@ module.exports = class AddGenerator extends Generator {
 			.then(() => {
 				if (action === "entry") {
 					return this.prompt([
-						Confirm("entryType", "Will your application have multiple bundles?")
+						Confirm("entryType", "Will your application have multiple bundles?"),
 					])
-						.then(entryTypeAnswer => {
+						.then((entryTypeAnswer: {
+							entryType: boolean,
+						}) => {
 							// Ask different questions for entry points
 							return entryQuestions(self, entryTypeAnswer);
 						})
-						.then(entryOptions => {
+						.then((entryOptions: object) => {
```

Comment:
You should be consistent with the types like you have done on previous functions